ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -350,6 +353,7 @@ export interface WorkspaceEditState {
   loading: boolean;
   showUnderservedPopulationDetails: boolean;
   showStigmatizationDetails: boolean;
+  reviewRequestedHasSelection: boolean;  // false initially, set to true once the user has chosen (either Yes or No)
```

Comment:
This is a Required Yes/No selection which is initially left blank

Reply:
another way would be to make the button display condition an enum like [true, false, undefined].  However we would still need a separate `reviewRequested` state because we pass that directly to an API